### PR TITLE
Fix various issues in the writing guidelines

### PR DESCRIPTION
**Pull request by npentrel**
While going through the writing guidelines I found a few formatting and other issues.

### CODE REVIEW

File: source/style-guide/writing/use-active-voice.txt [hunk 1]
```diff
@@ -70,18 +70,18 @@ of the user from those of the technology.
 
 .. _use-passive-voice:
 
-When Do You Use Passive Voice?
-------------------------------
+When To Use Passive Voice?
```
Comment (atsansone) [inline source/style-guide/writing/use-active-voice.txt:73]: **issue:** These headings are all set as questions to keep the language informal. If that's not preferred, let'd do it everywhere. I would revert this change.

File: source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt [hunk 2]
```diff
@@ -63,11 +63,12 @@ Use a Consistent Sentence Structure
 Restrict Sentence Length
 ------------------------
 
-A well-written long sentence may be hard to follow and understand.
+Even a well-written long sentence may be hard to follow and understand.
```
Comment (atsansone) [inline source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt:66]: **question:** Does adding "Even" aid in the reader's understanding? I would posit that it's extraneous.

File: source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt [hunk 3]
```diff
@@ -166,8 +167,8 @@ nominalization and use it instead.
 
 .. _use-necessary-words:
 
-Use the Necessary Words
------------------------
+Use Only Necessary Words
```
Comment (atsansone) [inline source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt:170]: **thanks:** Yes, this makes sense.

File: source/style-guide/writing/write-for-accessibility.txt [hunk 4]
```diff
@@ -80,7 +80,7 @@ Write for Ease of Reading
 
 - Break up walls of text to aid in scannability. For example, separate
   paragraphs, create headings, and use lists.
-- Use shorter sentences. Try to use fewer than 26 words per sentence.
+- Use shorter sentences. Try to use fewer than 25 words per sentence.
```
Comment (atsansone) [inline source/style-guide/writing/write-for-accessibility.txt:83]: **issue:** As originally written, this would be 25 or fewer words. With this change, it would be 24 or fewer words. For clarity, maybe rephrase to "25 or fewer words"?

**Review by atsansone — CHANGES_REQUESTED** [review]
@npentrel : A few small questions for you.

File: source/style-guide/writing/write-for-accessibility.txt (hunk 4 above)
Comment (npentrel) [reply source/style-guide/writing/write-for-accessibility.txt:83]: true enough. I'll revert this - though it sounds strange to me.

File: source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt (hunk 2 above)
Comment (npentrel) [reply source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt:66]: I had to re-read this a few times. The juxtaposition of well-written and hard to follow and understand didn't make sense to me at first. An alternative re-write would be `A well-written long sentence may still be hard to follow and understand`. Otherwise why have `well-written` in there at all? We could just say `Long sentences may be hard to follow and understand` but I think the meaning we are aiming for is that even if the long sentence is well-written, we'd still prefer shorter sentences.

File: source/style-guide/writing/use-active-voice.txt (hunk 1 above)
Comment (npentrel) [reply source/style-guide/writing/use-active-voice.txt:73]: How would you feel about `When Should You Use Passive Voice?`. My main complaint is that this page is an imperative "Use Active Voice" but we stop short of directly saying that there are cases when passive voice is admissible.

File: source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt (hunk 2 above)
Comment (atsansone) [reply source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt:66]: **comment:** Keep "Even" then. I get the distinction now.

File: source/style-guide/writing/use-active-voice.txt (hunk 1 above)
Comment (atsansone) [reply source/style-guide/writing/use-active-voice.txt:73]: **praise:** Perfect! I get the distinction and it makes more sense with `...Should You Use...`.

**Review by atsansone — APPROVED** [review]
@npentrel : LGTM! Thanks for this work!

**Comment by npentrel** [comment]
Great! I've made the changes (in a separate commit for clarity) - let me know if you want them squashed before merging - I guess you probably do that in the UI?

**Review by davemungo — APPROVED** [review]
LGTM
